### DISCUSSION TITLE
无法播放影片，报BadRequest:driveId, fileId cannot be empty

### DISCUSSION THREAD

**Discussion by chenhuiba** [AlistGo/alist#7276] [General]
播放影片时BadRequest:driveId, fileId cannot be empty.
查看日志：2024/9/27 22:39:39  09-27-2024 22:39:39 获取新file id失败:  <nil>
使用本地播放器：播放异常
folder_type.txt配置文件 ：r
阿里云盘路径：全部文件>备份文件›小雅转存文件夹

**Comment by seafront12** [comment]
一样的问题。找到解决办法了吗？
